cds_books/migrator/api.py

```diff
@@ -150,9 +162,11 @@ def get_multipart_by_legacy_recid(recid):
         ]
     )
     result = search.execute()
-    if result.hits.total < 1:
-        raise MultipartMigrationError(
-            'no multipart found with legacy recid {}'.format(recid))
+    if not result.hits or result.hits.total < 1:
+        click.secho('no multipart found with legacy recid {}'.format(recid),
+                    fg='red')
+        # raise MultipartMigrationError(
```

Comment:
nit: Is this used? is it a test? no commented code.

Reply:
This is not a final form and I left the comment to remember to change it when we have cleaner data. Otherwise I can't run the scripts and test relations on a bigger set of input because everything fails. Are there better ideas how to remember about it rather that leaving commented code?